fem/fespace.cpp

```diff
@@ -1525,6 +1525,67 @@ SparseMatrix *FiniteElementSpace::RefinementMatrix_main(
    return P;
 }
 
+SparseMatrix *FiniteElementSpace::VariableOrderRefinementMatrix_main(
```

Comment:
Why is this named `_main`? There's no other variant right?

Reply:
good point! Fixed.